content/en/docs/concepts/architecture/nodes.md

```diff
@@ -229,35 +229,35 @@ provider if the VM for that node is still available. If not, the node
 controller deletes the node from its list of nodes.
 
 The third is monitoring the nodes' health. The node controller is
-responsible for updating the NodeReady condition of NodeStatus to
-ConditionUnknown when a node becomes unreachable (i.e. the node controller stops
+responsible for updating the `NodeReady` condition of `NodeStatus` to
+`ConditionUnknown` when a node becomes unreachable (i.e. the node controller stops
 receiving heartbeats for some reason, for example due to the node being down), and then later evicting
 all the pods from the node (using graceful termination) if the node continues
 to be unreachable. (The default timeouts are 40s to start reporting
-ConditionUnknown and 5m after that to start evicting pods.) The node controller
+`ConditionUnknown` and 5m after that to start evicting pods.) The node controller
 checks the state of each node every `--node-monitor-period` seconds.
 
 #### Heartbeats
 
 Heartbeats, sent by Kubernetes nodes, help determine the availability of a node.
 
 There are two forms of heartbeats: updates of `NodeStatus` and the
-[Lease object](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/#lease-v1-coordination-k8s-io).
-Each Node has an associated Lease object in the `kube-node-lease`
+[`Lease` object](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/#lease-v1-coordination-k8s-io).
+Each node has an associated `Lease` object in the `kube-node-lease`
```

Comment:
What do you think about changing to:
Each `Node` object has an associate `Lease` object ...

Reply:
```suggestion
Each `Node` has an associated `Lease`, in the `kube-node-lease`
```